<!DOCTYPE html>
<html class="no-js" lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<title>Time for an interim review of Plazi’s Covid-19 related activities - Plazi</title>
	<script>(function(d,e){d[e]=d[e].replace("no-js","js");})(document.documentElement,"className");</script>
	<meta name="description" content="Three months ago we started with Pensoft an effort (see also press release) to contribute to a better understanding of the SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) aka corona virus that led to the global...">

	<link rel="stylesheet" href="/css/style.css">
	

	<link rel="shortcut icon" href="/img/favicon_io/favicon.ico">
	<link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
		integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A=="
		crossorigin=""/>

	<!-- the Leaflet js should be AFTER Leaflet's CSS -->
	<script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
		integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
		crossorigin=""></script>

	<link rel="stylesheet" href="/css/plazi_styles.css">
	<link rel="stylesheet" href="/css/expanding-search.css">

	
</head>
<body class="body">
	<div class="container container--outer">
		<header class="header">
	<div class="container header__container">
		
	<div class="logo logo--mixed">
		
			<div class="logo__item logo__imagebox">
					<a class="logo__link" href="/" title="Plazi" rel="home">
						<img class="logo__img" src="/img/logo-main.png" alt="Plazi">
					</a>
				</div>
		

		
		<div class="search-box">
			<input id="searchInput" type="text" class="search-input" placeholder="search…">
			
			<a class="search-btn" href="#"></a>
			<ul id="searchResults"></ul>
		</div>
	</div>
		
<!-- start: layouts/partials/menu.html -->
<nav class="menu">
	<button class="menu__btn" aria-haspopup="true" aria-expanded="false" tabindex="0">
		<span class="menu__btn-title" tabindex="-1">Menu</span>
	</button>
	<ul class="menu__list">
			<li class="menu__item">
				<a class="menu__link" href="/treatmentbank/"><span class="menu__text">treatmentbank</span></a>
			</li>
			<li class="menu__item">
				<a class="menu__link" href="/blr/"><span class="menu__text">BLR</span></a>
			</li>
			<li class="menu__item">
				<a class="menu__link" href="/services/"><span class="menu__text">Services</span></a>
			</li>
			<li class="menu__item">
				<a class="menu__link" href="/how-to-participate/"><span class="menu__text">Participate</span></a>
			</li>
			<li class="menu__item">
				<a class="menu__link" href="/ocellus/"><span class="menu__text">Image Search</span></a>
			</li>
			<li class="menu__item">
				<a class="menu__link" href="/litsearch/"><span class="menu__text">Lit Search</span></a>
			</li>
			<li class="menu__item">
				<a class="menu__link" href="/about/"><span class="menu__text">About</span></a>
			</li>
	</ul>
</nav>
<!-- end: layouts/partials/menu.html -->

	</div>
</header>
		<div class="wrapper flex">
			<div class="primary">
			
<main class="main" role="main">
	<article class="post">
		<header class="post__header">
			<h1 class="post__title">Time for an interim review of Plazi’s Covid-19 related activities</h1>
            <div class="meta__item-datetime">
                <svg class="meta__icon icon icon-time" width="16" height="14" viewBox="0 0 30 28"><path d="M15 0C7 0 1 6 1 14s6 14 14 14 14-6 14-14S23 0 15 0zm0 25C9 25 4 20 4 14S9 3 15 3s11 5 11 11-5 11-11 11zm1-18h-2v8.4l6.8 4.4L22 18l-6-3.8V7z"/></svg><time class="meta__text" datetime="2020-07-17T18:18:13&#43;02:00">July 17, 2020</time>
            </div>

			
		</header><div class="content post__content clearfix">
				<p><img src="img/custom-metadata.png" alt="Custom Metadata"></p>
<p>Three months ago we started with Pensoft an <a href="http://plazi.org/news/beitrag/plazi-and-pensoft-launch-an-initiative-to-provide-access-to-scholarly-published-data-about-coronavirus-hosts/a842a012361dd7a7c96b87e21bce8653/">effort</a> (see also <a href="https://blog.pensoft.net/2020/04/07/plazi-and-pensoft-join-forces-to-let-biodiversity-knowledge-of-coronaviruses-hosts-out/">press release</a>) to contribute to a better understanding of the SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) aka corona virus that led to the global COVID-19 pandemic, and in the followup joined the <a href="https://cetaf.org/news/joint-cetaf-dissco-covid-19-task-force-call-contributions">CETAF-Covid-19 task force</a>. With the end of the three months long taskforce we present here our view. A <a href="https://www.youtube.com/watch?v=Gnywe5hiMLU&amp;feature=youtu.be&amp;t=4622">multimedia presentation</a> by the knowledge base task force sub group has been streamed on July 17.</p>
<p><strong>Access to data in bat related biodiversity publications</strong></p>
<p>We started our effort under the assumptions that taxonomic knowledge of bats, the main known source of zoonotic virus spill-overs to humans, is reasonably well documented. Liberating treatments and figures from digital copies of bat literature was thus our initial aim. The situation, however, turned out to be rather different.</p>
<p>The taxonomy of bats is, in fact, not up-to-date on the main taxonomic name services. The <a href="http://www.catalogueoflife.org/col/browse/tree/id/69a5fd36318f9d0a471b8b45cf3d3a47">Catalogue of Life</a> was missing many taxa and synonyms. <a href="https://batnames.org/">Batnames.org</a>, the community resource for bat research, had an online list, but to get the entire list, including synonyms, required contacting the authors first. <a href="https://www.departments.bucknell.edu/biology/resources/msw3/">Mammal Species of the World</a> had an online list of bat species, including subspecies that could be downloaded as a <a href="https://www.departments.bucknell.edu/biology/resources/msw3/export.asp">MS-Excel spreadsheet</a>, an excerpt of the <a href="https://jhupbooks.press.jhu.edu/title/mammal-species-world">published catalogue</a>, and eventually the names would link to the <a href="https://www.departments.bucknell.edu/biology/resources/msw3/documents/MSW3Citations.pdf">PDF</a> including the cited references. The most recent catalogue, the <a href="https://www.lynxeds.com/product/handbook-of-the-mammals-of-the-world-volume-9/">volume 9, Chiroptera</a> in the Handbook of the Mammals of the World was published in 2019 as a book only priced at EUR 160, came with a CD-ROM of the cited literature. One section covered biology, another taxonomy. All these resources were short of changes of status of names, and publications augmenting the knowledge about a taxon.</p>
<p>Access to literature was only to some extent available, often only indirectly by the combination of taxon authority and year (e.g. <em>Rhinolophus sinicus</em> K. Andersen, 1905), at best as a bibliographic reference hardly ever complemented with a digital object identifier (DOI) or a link to a digital copy of the referenced work.</p>
<p>With this barrier, access to biotic interactions, behavioral traits such as co-roosting as source for lateral transmission of viruses, predator-prey relationships, a main use case by our task force is seriously limited, invoking a major effort to advance. Access to literature was furthermore complicated by the confinement which led to a separation of the scientists from their resources, such as collections of PDFs or hard copies.</p>
<p>The first conclusion thus is that at the launch of the task force work, much more time for preparation to data liberation, such as creating a catalogue of bats with extended bibliographic references as starting point to finding the original publications, and for that finding and contacting the appropriate scientists was needed. Even for groups with well curated catalogues, the step from a referenced bibliographic record to have all the digital copies at hand comes at a great cost.</p>
<p><strong>The CETAF task force as an ad hoc group</strong></p>
<p>The formation of the <a href="https://cetaf.org/news/joint-cetaf-dissco-covid-19-task-force-call-contributions">CETAF Covid-19 task force</a> we joined early on, has been a very unusual experience. The very wide expertise in the group, the curiosity to find out what the other members do, and sharing of resources led to a <a href="https://www.youtube.com/watch?time_continue=1&amp;v=60KMgjqhP-E&amp;feature=emb_logo">very constructive team</a>, as part of an equally inspiring task force membership. The expertise included taxonomy, ecology, biodiversity informatics, and skills from liberating data, cataloguing, building repositories, reuse of data in <a href="https://www.gbif.org/">GBIF</a> and <a href="https://www.globalbioticinteractions.org/">GloBI</a> to scientific analyses.</p>
<p>Aspects of the virus spill-over has been the guiding theme. Discussing and implementing the input workflow to answer respective research questions, being confronted with all the pitfalls, inspired and helped the creation of prospective publishing tools by <a href="https://pensoft.net/">Pensoft</a>, <a href="https://www.globalbioticinteractions.org/">GloBI</a> and Plazi. The aim was to publish biotic interactions in a format that can be reused by machine without any further semantic conversion. It also inspired the insertion of biotic interactions specific metadata at <a href="https://zenodo.org/search?page=1&amp;size=20&amp;q=keywords:%22CETAF-taskforce%22">Zenodo</a>. This allows annotating relevant publications not only with the host and virus names mentioned in the article, but also with the specific interactions needed for further analyses.</p>
<p>The second conclusion is that the circumstances led to an ad hoc team of scientists that openly shared data, expertise and time that exceeded what research team normally has. This setup advanced substantially the knowledge of the biology of the bat virus interactions, with implications well beyond the theme of the task force itself.</p>
<p><strong>The role of taxonomic names</strong></p>
<p>Taxonomic names are one way to organize the knowledge of biodiversity. Over time and with increasing knowledge, the names associated with taxon concepts can change, covering more or less inclusive concepts. These are based on specimens, a.k.a. vouchers, which would not necessarily change their names in due course, especially in the databases. Thus there is a substantial amount of data available which will not be discovered with the names in use in 2020 because the links to previous name usages are not available, or because specimens have been misidentified.</p>
<p>The taxonomic literature has a convention to cite previous taxonomic treatments and type the relationship, making it explicit when a name changes, and explaining at the same time why. This relationship can be tagged and are at the base of <a href="https://synospecies.plazi.org/">Synospecies</a>, a tool based on facts (RDF triples) in a triple store provided by <a href="https://franz.com/">Franz</a>. This service can provide the current status of a taxonomic name and all its alternatives, for example for the main bat focus species in Covid-19, <a href="https://synospecies.plazi.org/#Rhinolophus+sinicus">Rhinolophus sinicus Andersen, 1905</a>. Having the link to the treatment allows access to the data a scientist used to propose, augment or deprecate a taxon.</p>
<p>For bats, 1412 treatments covering 1072 taxonomic names, 505 treatment citations, and 63 new species from 95 publications are now available on <a href="http://tb.plazi.org/GgServer/srsStats/stats?outputFields=tax.name+tax.orderEpithet+tax.status+cit.treatCitCount&amp;groupingFields=tax.orderEpithet+tax.status&amp;FP-tax.orderEpithet=Chiroptera&amp;format=HTML">Plazi</a>. It also allows to link cited specimens in a treatment to the actual specimen and potentially to link a specimen to the treatments where it has been used.</p>
<p>The third conclusion is that we have new, complementary tools to mine and visualize data in the biodiversity literature, such as treatments and treatment citations as baseline for the catalogue of life and at the same time provide access to all the data about the taxa.</p>
<p><strong>Access to and data extraction</strong></p>
<p>Our tool and service to mine biodiversity literature is the Plazi data liberation workflow. Its first step is preparing the respective literature for machine conversion. Two bat specific resources <a href="https://batnames.org/">batnames.org</a> and <a href="https://jhupbooks.press.jhu.edu/title/mammal-species-world">Mammal Species of the World</a> were online accessible, whereas all the other previous catalogues only exist in printed format. As a consequence, we purchased printed copies of the catalogues, cut the spine of the books off, scanned, OCRed and deposited a digital copy in the Biodiversity Literature Repository and the Corona Host Virus community at Zenodo (e.g. <a href="https://doi.org/10.5281/zenodo.3873244">Honacki et al. 1982</a>, <a href="https://doi.org/10.5281/zenodo.3875153">Corbet &amp; Hill, 1991</a>; <a href="https://doi.org/10.5281/zenodo.3748525">Burgin, 2019</a>). The Dutch digitization company <a href="https://picturae.com/nl/">Picturae</a> supported the work by scanning for us our purchased Bats catalogues.</p>
<p>Another source has been <a href="https://bioone.org/">BioOne</a> and <a href="https://bioone.org/journals/acta-chiropterologica/author-guidelines">Acta Chiropterologica</a> which agreed to collaborate to explore and provide access to data in their bat specific scientific journal. The conversion of 44 articles liberated 552 treatments, 401 figures, 26 new species (<a href="http://tb.plazi.org/GgServer/dioStats/stats?outputFields=doc.articleUuid+bib.source+cont.pageCount+cont.treatCount+cont.treatCitCount+cont.matCitCount+cont.figCount+cont.tabCount+cont.bibRefCitCount&amp;groupingFields=bib.source&amp;FP-bib.source=Acta%25Chiro%25&amp;format=HTML">Summary</a>; <a href="http://tb.plazi.org/GgServer/srsStats/stats?outputFields=doc.uuid+bib.author+bib.title+bib.year+bib.source+tax.name+tax.status+cit.treatCitCount+cit.figCitCount&amp;groupingFields=doc.uuid+bib.author+bib.title+bib.year+bib.source+tax.name+tax.status+cit.treatCitCount+cit.figCitCount&amp;orderingFields=-tax.status&amp;FP-bib.source=%22Acta%20Chiropterologica%22&amp;format=HTML">Detail</a>).</p>
<p>A third source has been a pre-processed copy of Linnaeus’ <a href="http://plazi.org/news/beitrag/the-taxonomic-treatments-from-linnaeus-systema-naturae-1758-10th-edition-liberated/8f2fef3ef0a651ac80768c66436b24d1/">Systema Naturae</a> from 1758, the starting point of modern taxonomy supplied by Richard Pyle from Zoobank. This converted copy originated from Dave Remsen who converted the original copy into text which then got refined by Richard Pyle (<a href="http://treatment.plazi.org/GgServer/summary/3F896AFC21C459639D0D71D9EDB8875C">TreatmentBank</a>; <a href="https://www.gbif.org/dataset/57ea6fdb-2533-44e5-b60b-736758a88d19">GBIF</a>)</p>
<p>A forth source has been from Gabor Csorba who provided an extensive collection of PDFs covering the target groups of Rhinolophidae and  Hipposideridae- These files will now be made available on the Biodiversity Literature Repository and Coronavirus Host communities at Zenodo, and chained up to be mined.</p>
<p>The processed publications (see <a href="http://tb.plazi.org/GgServer/srsStats/stats?outputFields=tax.name+tax.orderEpithet+mat.matCitCount&amp;groupingFields=tax.orderEpithet&amp;orderingFields=tax.name&amp;limit=100&amp;FP-tax.orderEpithet=Chiroptera&amp;format=HTML">stats</a>, <a href="http://tb.plazi.org/GgServer/srsStats/stats?outputFields=doc.uuid+tax.name+tax.orderEpithet+tax.familyEpithet+tax.status+mat.matCitCount&amp;groupingFields=doc.uuid+tax.name+tax.orderEpithet+tax.familyEpithet&amp;orderingFields=tax.name+-tax.familyEpithet&amp;limit=100&amp;FP-tax.orderEpithet=Chiroptera&amp;format=HTML">list</a>) are available on Plazi’s <a href="http://tb.plazi.org/GgServer/search">TreatmentBank</a> and the <a href="https://zenodo.org/communities/biosyslit/search?page=1&amp;size=20">Biodiversity Literature Repository</a>. They are accessible through <a href="http://plazi.org/api-tools/statistics/">Plazi stats</a>, or through more specific tools like <a href="https://ocellus.info/images.html?q=Chiroptera&amp;size=100&amp;page=1&amp;communities=biosyslit">Ocellus</a> for images and <a href="https://synospecies.plazi.org/">Synospecies</a> for names respectively. They are also available at GBIF and can easily be found, querying the DOI in the search box.</p>
<p>Non-taxonomic literature including data on virus host relationship has been collected starting with the most recent review and research publications and following the cited publications. They have been uploaded to a staging area in <a href="https://www.zotero.org/groups/2470546/plazi_coronavirus_virus-hosts_library/library">Zotero</a> and manually attributed with the host, virus and host-virus relationship and finally uploaded to <a href="https://zenodo.org/search?page=1&amp;size=20&amp;q=keywords:%22CETAF-taskforce%22">BLR</a> and the <a href="https://zenodo.org/communities/coviho/search?page=1&amp;size=20&amp;q=Coronaviridae">covhiho</a> community at Zenodo. In the 160 articles 150 hosts, 247 viruses and 1,146 biotic interactions have been extracted.</p>
<p>The fourth conclusion is that even for such prominent taxa as bats are, taxonomic and biodiversity data are not readily accessible in a digital format. This access is very time-consuming to provide, and thus are largely absent in research. Even finding the publications requires quite some domain knowledge.</p>
<p>All the taxonomic treatments liberated are ultimately based on specimens, as physical specimens and more recently as genes or observations. This relationship can be very loose (Linnaeus refers to his collection) to be very specific by providing a list of materials citation including specimen codes to ultimately persistent identifiers. However, the infrastructure to find and obtain the respective identifiers, to a large extent, does not exist yet.</p>
<p>The seventh conclusion is that there is a gap between biodiversity research results and the raw data, that is the specimens. This is detrimental to biodiversity research because a lot of data and knowledge is omitted in situations like the Covid-19 pandemic or understanding future spill-overs.</p>
<p><strong>Data reuse and research data life cycle</strong></p>
<p>All the processed data are reused by <a href="https://www.gbif.org/publisher/7ce8aef0-9e92-11dc-8738-b8a03c50a862">GBIF</a> as data sets. Through this, as a beginning, the chapters on <a href="https://www.gbif.org/dataset/6384b520-7e9f-4874-a414-76c2e9b01d74">Rhinolophidae</a> and <a href="https://www.gbif.org/dataset/b7e93c79-44a6-4d9c-8b78-dcb62033fb0d">Hipposideridae</a> of the most recent catalogue of bats, the Handbook of the Mammals of the World Vol. 9, as well as the very first, Linnaeus’ <a href="https://www.gbif.org/publisher/7ce8aef0-9e92-11dc-8738-b8a03c50a862">Systema Naturae 1758</a> with all the treatments are available in their taxonomic backbone.</p>
<p>Together with GloBI and Quentin Groom&rsquo;s indexing, this data added to the 17,123 taxonomic names and 85,493 interactions to <a href="https://www.globalbioticinteractions.org/2020/05/26/releasing-biological-data-from-COVID-19-lockdown/">GloBI</a>. <a href="https://www.google.com/url?q=https://travis-ci.com/github/globalbioticinteractions/zenodo-metadata/jobs/361953200%23L203&amp;sa=D&amp;ust=1594965632386000&amp;usg=AFQjCNEjyHzrM8F5VpKmqYB_b39d1_caow">GloBI</a> indexed 138 of the publications on Zenodo.</p>
<p>The extracted bat names from this corpus of literature are being reused in the <a href="https://docs.google.com/spreadsheets/d/1Hddg2dwl_H6iyhVCY10qyV6gTyFHafc646DeWi5jntI/edit#gid=1391555866">CoVoc</a>, a vocabulary to mine literature in PubMed adding especially terms in regards of host and Corona virus relationships. To widen PubMed, a small project has been awarded to the <a href="https://www.sib.swiss/">Swiss Institute for Bioinformatics</a> (Patrick Ruch Swiss Institute of Bioinformatics, including Plazi) to include taxonomic treatments as an additional publication type for text and data mining. Together this will allow discovering more published host virus relationships over a wider corpus of publications.</p>
<p>Together with Pensoft and GloBI a <a href="https://blog.pensoft.net/2020/04/24/how-to-get-data-from-research-articles-back-into-the-research-cycle-%D0%B0t-no-additional-costs/">standard template</a> for appendices in Pensoft publications have been developed and implemented that allows direct harvesting and conversion of linked FAIR data, such as specimen codes, accession number of biotic interactions. An exemplar application is the publication by <a href="https://doi.org/10.3897/zookeys.929.50240">Patterson et al., 2020</a>.</p>
<p><strong>Infrastructure development</strong></p>
<p>Metadata in publications deposits allows you to find more efficiently target publications. Together with Jorrit Poelen (<a href="https://www.globalbioticinteractions.org/">GloBI</a>) and Alex Ioannids from Zenodo, custom metadata for host, virus and their specific relationships reported in the respective publication have been added to <a href="https://zenodo.org/search?page=1&amp;size=20&amp;q=keywords:%22CETAF-taskforce%22">Zenodo</a>, see e.g. <a href="https://zenodo.org/record/3949199">Lau et al.</a>. The metadata is a subset of and references the Molecular Interactions Controlled Vocabulary.  This allows GloBi - and in fact anybody - to index this corpus of literature and <a href="https://www.globalbioticinteractions.org/browse/?accordingTo=globi%3Aglobalbioticinteractions%2Fzenodo-metadata&amp;interactionType=interactsWith&amp;resultType=json">reuse it</a>. This is the second set of custom metadata implemented in Zenodo for biodiversity deposits, the former covering geo-coordinate pairs for observations listed in taxonomic treatment deposits. This development is using Zenodo’s  potential as a general repository to allow customization for specific communities.</p>
<p>Making increasingly complex and larger corpora needs a high degree of automation, which is possible via the <a href="https://developers.zenodo.org/">Zenodo’s API</a>. Together with <a href="https://zenodo.org/">Zenodo</a>, their infrastructure  has been extended to deal with custom metadata, in this case for the biotic interactions.</p>
<p><strong>The future of publishing</strong></p>
<p>As the above shows, published knowledge locked into opaque PDF documents is not helpful so let’s turn publications into structured, searchable knowledge graphs. To respond quickly to new challenges the publications have to be based on the open access <a href="https://www.force11.org/group/fairgroup/fairprinciples">FAIR principles</a>, implicit links to cited material have to be explicit, and the data therein has to be usable by machine.</p>
<p>Having the chance to work with publishers in the task force provides the opportunity to implement obvious changes to the existing publications system. The most obvious is to cut down as much as possible parts of or the entire <a href="http://dx.doi.org/10.1186/1756-0500-2-53">Plazi workflow</a> from finding to processing publications to liberate data. Though there are hundreds of millions of pages to process - including new pages every day - the only way forwards is to use semantic publishing. A big advancement in this task force has been the development of publications that include publishing data that can immediately be harvested and reused. The proof is the publishing of biotic interactions and indexing by GloBI (see <a href="https://doi.org/10.3897/zookeys.929.50240">publication</a>, reused interaction)</p>
<p><strong>What’s after the CETAF Covid-19 task force?</strong></p>
<p>The sheer magnitude of the unavailability of published research data is a daunting task, coupled with very limited funding for converting legacy publications into data. Data that is <em>nota bene</em> the foundation to improve the understanding of the dynamics of virus spill-over.</p>
<p>Even within the task force group, there is insufficient awareness of the big data - tens of billions of facts - hidden in unstructured publications and the fledgling tools to extract, store, explore, cite and reuse this data. Research projects as proposed and started in the task force on virus spill-over will be the best communication platform. We will spend additional effort to communicate and build the needed awareness.</p>
<p>We will continue to explore funding possibilities for research projects formulated within the taskforce that integrate research questions and legacy literature.</p>
<p>We will make use of the momentum and extract all the treatments in the main mammal taxonomic catalogues to provide to the community access to the mammal classifications provided by the standards at the given time, and which is often the reference for name usage in ecological studies or the identification of specimens in collections that end up in GBIF. We believe that this might serve as a very good example of the value of having entire catalogus and their treatments available. It will also show the difference from having a name to being able to know what’s behind it.</p>
<p>These data liberation operations will be maintained thanks to the support of  <a href="https://www.arcadiafund.org.uk/">Arcadia</a>. Their unique, generous support is instrumental to change the way taxonomic data is shared and accessible and to build underlying infrastructure together with Zenodo that make this change possible and sustainable.</p>
<p>Finally our lesson learned, once more, is to invest in collaborations with publishers, specifically <a href="https://pensoft.net/">Pensoft</a>,  CETAF-<a href="https://europeanjournaloftaxonomy.eu/index.php/ejt/index">European Journal of Taxonomy</a>, the <a href="http://sciencepress.mnhn.fr/fr">Museum National d’Histoire Naturelle</a>, Paris publishing and <a href="https://bioone.org/">BioOne</a> to transition in an area where biodiversity data is from its inception digitally accessible and reused by aggregators like <a href="https://www.gbif.org/">GBIF</a> or <a href="https://elixir-europe.org/">ELIXIR</a>.</p>
<p>We are looking forward to more task forces of this inspiring kind.</p>

		</div>
		<footer class="post__footer">
			
<div class="post__tags tags clearfix">
	<svg class="tags__badge icon icon-tag" width="16" height="16" viewBox="0 0 32 32"><path d="M32 19c0 1-1 2-1 2L21 31s-1 1-2 1-2-1-2-1L2 16c-1-1-1.4-2-1.4-2S0 12.5 0 11V3C0 1.5.8.8.8.8S1.5 0 3 0h8c1.5 0 3 .6 3 .6S15 1 16 2l15 15s1 1 1 2zM7 10a3 3 0 1 0 0-6 3 3 0 0 0 0 6z"/></svg>
	<ul class="tags__list">
		<li class="tags__item">
			<a class="tags__link btn" href="/tags/news/" rel="tag">news</a>
		</li>
	</ul>
</div>
		</footer>
	</article>
</main>




			</div>
			
		</div>
		<footer class="footer">
	<div class="container footer__container flex">
		
<div class="footer__links">
	<a class="footer__link" href="https://www.eurekalert.org/news-releases/952793">Supported in part by Arcadia</a> | <a class="footer__link" href="/privacy/">Privacy</a> | <a class="footer__link" href="/disclaimer/">Legal Disclaimer</a>
</div>
		<div class="footer__copyright">
			<span class="footer__copyright-credits">&copy; 2025 Plazi. Published under <a href="https://creativecommons.org/publicdomain/zero/1.0/" target="_blank">CC0 Public Domain Dedication</a></span>
			
		</div>
	</div>
</footer>
	</div>

	
	<script src="https://cdn.jsdelivr.net/npm/minisearch@6.0.1/dist/umd/index.min.js"></script>
	<script src="/js/search-mini.js"></script>

	<script async defer src="/js/menu.js"></script>

	</body>
</html>